src/main/java/com/refinedmods/refinedstorage/apiimpl/network/grid/CraftingGridBehavior.java

```diff
@@ -24,7 +25,7 @@
 
 public class CraftingGridBehavior implements ICraftingGridBehavior {
     @Override
-    public void onCrafted(INetworkAwareGrid grid, ICraftingRecipe recipe, PlayerEntity player) {
+    public void onCrafted(INetworkAwareGrid grid, ICraftingRecipe recipe, PlayerEntity player, IStackList<ItemStack> networkCraftingItems, IStackList<ItemStack> extractedItems) {
```

Comment:
We should probably mark these 2 params with `@Nullable`

Reply:
Renaming `networkCraftingItems` to `availableItems` is probably better.

Reply:
`extractedItems` is slightly confusing since we didn't actually extract. `usedItems` perhaps?